content/shmem_put.tex

```diff
@@ -56,7 +56,7 @@
     The routines return after the data has been copied out of the \source{} array
     on the local \ac{PE}.  The delivery of data words into the data object on the
     destination \ac{PE} may occur in any order.  Furthermore, two successive put
-    routines may deliver data out of order unless a call to \FUNC{shmem\_fence} is
+    routines may deliver data out of order, unless a call to \FUNC{shmem\_fence} is
```

Comment:
This is a misapplication of _Comma Use No. 2_ in [Purdue's Extended Rules for Using Commas](https://owl.english.purdue.edu/owl/resource/607/02/) for a dependent (subordinate) clause following the main clause.